docs/troubleshooting.md

```diff
@@ -0,0 +1,48 @@
+# Issues we met
+
+This document collects issues that we had with sssd-test-suite and their
+solution.
+
+## Ad machine sudden shutdown
+Probably license expired. Run
+
+    ./sssd-test-suite up -s ad ad-child && ./sssd-test-suite provision rearm
+
+
+## TASK [host : Install required vagrant plugins] failed due to vagrant dependencies
+
+remove ~/.vagrant.d
+uninstall vagrant-sshfs & vagrant-libvirt rpms and rerun host provisioning
+
+    rm -rf ~/.vagrant.d
+    sudo dnf remove vagrant-sshfs vagrant-libvirt
+    ./sssd-test-suite provision host -p "$POOL_DIR"
+
+
+## Various errors on fc28
+sssd-test-suite needs new ansibles, please upgrade to fc30.
```

Comment:
Perhaps saying than it keeps it is under active development so if there are any issue try to upgrade ansible?